modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -643,9 +649,18 @@ namespace gapi {
          * @param rhs "Right-hand-side" package in the process
          * @return a new kernel package.
          */
-        friend GAPI_EXPORTS GKernelPackage combine(const GKernelPackage  &lhs,
+        friend GAPI_EXPORTS cv::GKernelPackage cv::gapi::combine(const GKernelPackage  &lhs,
```

Comment:
and here, as this friend declaration is **actually inside** `GKernelPackage` class, I don't think `cv::` is needed for `cv::GKernelPackage`

Reply:
Applied